website_product_share/README.rst

```diff
@@ -0,0 +1,57 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Product Share
+=============================
+
+Allow user to enable/disable share button on each product page
```

Comment:
Then the redaction of this should change a bit, because it's a bit confussing. @elicoidal, can you help us with your redaction skills?